maya/core.py

```diff
@@ -806,4 +806,6 @@ def intervals(start, end, interval):
     while current_timestamp.epoch < end.epoch:
         yield current_timestamp
 
-        current_timestamp = current_timestamp.add(seconds=interval.seconds)
+        current_timestamp = current_timestamp.add(
+            seconds=interval.total_seconds()
```

Comment:
I think this is the right solution. 
I don't have a strong grasp on how microseconds interact with maya internals, so it's possible that a better solution here would be calculating number of seconds manually: `(td.seconds + td.days * 24 * 3600)` per the Python 2 docs. From reading the docs (linked below) it looks like accounting for microseconds causes us to lose accuracy over a timedelta of 270 years or so.

Python 2 docs: https://docs.python.org/2/library/datetime.html#datetime.timedelta.total_seconds
Python 3 docs: https://docs.python.org/3/library/datetime.html#datetime.timedelta.total_seconds

Reply:
Looks good to me! :tada: